web/ts/create-index.ts

```diff
@@ -1,17 +1,15 @@
-const MiniSearch = require('minisearch');
 import { promises as fs } from 'fs';
 import * as path from 'path';
+const MiniSearch = require('minisearch');
+const MiniSearchConfig = require('../src/minisearch.config');
+
 const { StringDecoder } = require('string_decoder');
 const decoder = new StringDecoder('utf8');
 
-const DATADIR = '../../data';
+const DATADIR = '../public/data';
```

Comment:
process.argv?

Reply:
then you'd need to maintain a shell script or a make file to run it...